packages/testing/src/index.ts

```diff
@@ -0,0 +1,157 @@
+import * as activity from '@temporalio/activity';
+import { AsyncCompletionClient, Connection, WorkflowClient } from '@temporalio/client';
+import { ActivityFunction, CancelledFailure } from '@temporalio/common';
+import { Core } from '@temporalio/worker';
+import path from 'path';
+import { AbortController } from 'abort-controller';
+import { ChildProcess, spawn } from 'child_process';
+import events from 'events';
+import { kill, waitOnChild } from './child-process';
+import type getPortType from 'get-port';
+
+/**
+ * Options for {@link TestWorkflowEnvironment.create}
+ */
+interface TestWorkflowEnvironmentOptions {
+  testServerSpawner?(port: number): ChildProcess;
+}
+
+function defaultTestWorkflowEnvironmentOptions(): Required<TestWorkflowEnvironmentOptions> {
+  return {
+    testServerSpawner(port: number) {
+      return spawn(path.join(__dirname, '../test-server'), [`${port}`], {
+        stdio: 'ignore',
+      });
+    },
+  };
+}
+
+// TS transforms `import` statements into `require`s, this is a workaround until
```

Comment:
What's wrong with require?

Reply:
Doesn't work, at some point sindresorhus decided he's making all of his packages ESM and you have to `import` them.